omnibus/.kitchen.yml

```diff
@@ -11,7 +11,7 @@ driver:
     cpus: 4
     memory: 4096
   synced_folders:
-    - ['../..', '/home/vagrant']
```

Comment:
The previous version of this mounted chef-dk at /home/vagrant/chef-dk, and ../omnibus at /home/vagrant/omnibus, etc. I think we still want that, don't we?